File: src/kyu_8/multiply_the_number/js/multiplyTest.js
import multiply from "./multiply";

describe("Basic Tests",() =>{
    Test.assertEquals(multiply(10),250);
    Test.assertEquals(multiply(5),25);
    Test.assertEquals(multiply(200),25000);
    Test.assertEquals(multiply(0),0);
    Test.assertEquals(multiply(-2),-10);
})

File: src/kyu_8/multiply_the_number/python/multiply_test.py
import codewars_test as test
from solution import multiply

@test.describe("Fixed Tests")
def fixed_tests():
    @test.it('Basic Test Cases')
    def basic_test_cases():
        test.assert_equals(multiply(10),250)
        test.assert_equals(multiply(5),25)
        test.assert_equals(multiply(200),25000)
        test.assert_equals(multiply(0),0)
        test.assert_equals(multiply(-2),-10)